test/functional/apps/settings/_index_pattern_results_sort.js

```diff
@@ -93,9 +93,11 @@ define(function (require) {
         });
 
         bdd.it('makelogs data should have expected number of fields', function () {
-          return settingsPage.getFieldsTabCount()
-          .then(function (tabCount) {
-            expect(tabCount).to.be('' + expectedFieldCount);
+          return common.tryForTime(5000, function () {
```

Comment:
The problem is that setFindTimeout does wait to find the tab element, but then tries to getVisibleText and fails with a stale element exception because that text is being written between the time we found the tab and the time it tries to get the text.
that's why I used a tryForTime around getting the expected value
the other ways to solve it are (1) a long sleep :-( or (2) waitForSpinnerDone (or something else that let's us know there's no page updates still in progress)